param/find_mono_energy.par

```diff
@@ -0,0 +1,10 @@
+arffile,f,a,"",,,"Input ARF file"
+rmffile,f,a,"",,,"Input RMF file"
+model,s,a,"xswabs.abs1*xspowerlaw.pwrlaw",,,"Sherpa model expression"
```

Comment:
The `xswabs` model is "old" and I wonder if we should use a more-modern equivalent (e.g. `xsphabs`). I don't expect it to make a significant difference as we are looking at significantly-larger scales than we see differences. For funsies

```
sherpa In [7]: xsphabs.phabs
-------------> xsphabs.phabs()
Out[7]: <XSphabs model instance 'xsphabs.phabs'>

sherpa In [8]: xswabs.wabs
-------------> xswabs.wabs()
Out[8]: <XSwabs model instance 'xswabs.wabs'>

sherpa In [9]: phabs.nh = 0.2

sherpa In [10]: wabs.nh = 0.2

sherpa In [11]: egrid = np.arange(0.1, 1, 0.01)

sherpa In [12]: yphabs = phabs(egrid[:-1], egrid[1:])

sherpa In [13]: ywabs = wabs(egrid[:-1], egrid[1:])

sherpa In [14]: emid = (egrid[:-1] + egrid[1:]) / 2

sherpa In [15]: plt.clf()

sherpa In [16]: plt.plot(emid, ywabs, label='wabs')
Out[16]: [<matplotlib.lines.Line2D at 0x7ff22022ee50>]

sherpa In [17]: plt.plot(emid, yphabs, label='phabs')
Out[17]: [<matplotlib.lines.Line2D at 0x7ff22036fe50>]

sherpa In [18]: plt.legend()
Out[18]: <matplotlib.legend.Legend at 0x7ff23001b670>

sherpa In [19]: plt.xlabel('Energy (keV)')
Out[19]: Text(0.5, 120.87222222222219, 'Energy (keV)')
```

![x](https://user-images.githubusercontent.com/224638/123662220-3d4ead80-d803-11eb-9bfd-24d3d99e36f7.png)